configure-buildfarm/tasks/main.yml

```diff
@@ -30,6 +37,26 @@
     dest: /tmp/jenkins-cli.jar
     mode: 0775
 
+-
+  name: "Create Groovy script from template"
+  template:
+    src: proxy-config.j2
+    dest: "/tmp/proxy-config.groovy"
+    force: yes
+  connection: local
+  register: proxy_task
+  when:
+    - proxy_host is defined
+    - proxy_host != ''
+    - proxy_port is defined
+    - proxy_port != ''
+
+-
+  name: "Configure HTTP Proxy Settings"
+  command: java -jar /tmp/jenkins-cli.jar -s https://{{ route_output.stdout }}  groovy /tmp/proxy-config.groovy
```

Comment:
We can use {{ jenkins_route_protocol }} here. This is now set in the `vars/buildfarm.yml` in productised. Could we add it in the community vars file also?